src/Raven.Server/Documents/Handlers/Admin/AdminConfigurationHandler.cs

```diff
@@ -72,6 +73,40 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        //TODO
+        [RavenAction("/databases/*/admin/configuration/settings2", "GET", AuthorizationStatus.DatabaseAdmin, IsDebugInformationEndpoint = true)]
```

Comment:
let's change that to `/databases/*/admin/record` and reuse code from `AdminDatabasesHandler`